tests/signal_manager_test.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+include './src/signal_manager.sh'
+include './tests/utils'
+
+function test_default_handler()
+{
+  local -r original_dir="$PWD"
+  local -r expected=$'\n'"Oh no! An interruption! See ya..."
+
+  cd "$SHUNIT_TMPDIR" || exit
```

Comment:
Hmmm... I'm not too fond of this... if we exit in case of an invalid directory, we will not run the other tests. I prefer something like:
```
cd "$SHUNIT_TMPDIR" || fail "($LINENO): Any good message"
```